shared/bazel/rules/cc_rules.bzl

```diff
@@ -7,6 +7,30 @@ load("@rules_cc//cc/common:cc_common.bzl", "cc_common")
 load("@rules_pkg//:mappings.bzl", "pkg_files")
 load("@rules_pkg//:pkg.bzl", "pkg_zip")
 
+# Coppied from bazel since it isn't exposed publicly that I can find.
```

Comment:
Where exactly was this copied from? (link please)

Reply:
Yup, done!